css/quotes.css

```diff
@@ -1,185 +1,271 @@
+/*For mobile phones: */
 body {
   margin: 0;
+  overflow-x: hidden;
 }
 
-.form {
-  margin-left: 5%;
-  margin-right: 5%;
+.form__container {
+  /* margin-left: 5%;
+  margin-right: 5%; */
   max-width: 90%;
```

Comment:
please remove commented rules, also if in the HTML you are using the generic `container` helper, what is the purpose of setting `max-width: 90%;`? where did the value of 90% come from?